tests/test_main.py

```diff
@@ -301,7 +301,7 @@ def test_dotenv_values_file(dotenv_file):
     [
         # Defined in environment, with and without interpolation
         ({"b": "c"}, "a=$b", False, {"a": "$b"}),
-        ({"b": "c"}, "a=$b", True, {"a": "$b"}),
+        ({"b": "c"}, "a=$b", True, {"a": "c"}),
```

Comment:
Note this is also a fix. Since the interpolation is set to `True`.
`$b` and `${b}` should have the same behavior.